src/utils/logger.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+
+def get_logger() -> logging.Logger:
+    """
+    Get the logger for the application and set it up
+    :return: logger
+    """
+    logger = logging.getLogger("lynx")
+    logger.setLevel(logging.DEBUG)

```

Comment:
it's too noisy with multiple agents running, please add some way (maybe cmdline args) to change the log level to INFO, WARNING, ...

Reply:
Logger level should be provided in program parameter, the available levels:
```
map_level = {
    "DEBUG": logging.DEBUG,
    "INFO": logging.INFO,
    "WARNING": logging.WARNING,
    "ERROR": logging.ERROR
}
```